src/catacharset.cpp

```diff
@@ -150,9 +150,13 @@ int utf8_width( const char *s, const bool ignore_tags )
             continue;
         }
         if( ignore_tags ) {
-            if( ch == '<' ) {
+            const char *const tag1 = "color_";
```

Comment:
It might be less performant, but it could make more sense to remove all the tags from the string.
Isn't `remove_color_tags` doing this actually?

Reply:
> the intent of the string comparisons.

For example, if we consider a string `"< <color_cyan>Map editor</color> >"`, and set `inside_tag` to true on every occurrence of `<` (the previous logic), the characters that will be counted for text width will only be `"Map editor >"`, without the starting bracket and space. If we check if `<` is followed by `"color_"` or `"/color>"`, then it will properly count the starting characters. As I said in the OP this is only a temporary solution, but considering the state of how tags are handled currently in the many output functions, a proper fix would be complicated and should probably go in another PR.

Reply:
I also think `remove_color_tags` would more clear here.

Reply:
It would, but `utf8_width` needs fixing regardless since it's also called in other functions such as `right_print`. Maybe change `utf8_width` to also use `remove_color_tags` if `ignore_tags` is true?